### PR TITLE
html-forms - Dennis Liu

### PR DESCRIPTION
**Pull request by dennisliu9**
html-forms_solution

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <label for="email">Email</label>
+    <div>
+      <input type="text" name="email">
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This `input` needs an `id` to match the `for` attribute of its `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    </div>
+    <label for="coolreason">Why is coding cool?</label>
+    <div>
+      <select name="coolreason">
```
Comment (codingcodymiller) [inline html-forms/index.html:15]: This `select` needs an `id` to match the `for` attribute of its `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      <input id="HTML" type="radio" name="favlang" value="html">
+      <label for="HTML">HTML</label>
+      <input id="CSS" type="radio" name="favlang" value="css">
+      <label for="CSS">CSS</label>
+      <input id="JavaScript" type="radio" name="favlang" value="javascript">
+      <label for="JavaScript">JavaScript</label>
+    </div>
+    <label for="signup">Would you like to sign up for more information?</label>
+    <div>
+      <input type="checkbox" name="signup" value="y">Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: "Sign me up!" should be the `label` here

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <p>Favorite Language (choose one)</p>
+    <div>
+      <input id="HTML" type="radio" name="favlang" value="html">
+      <label for="HTML">HTML</label>
+      <input id="CSS" type="radio" name="favlang" value="css">
+      <label for="CSS">CSS</label>
+      <input id="JavaScript" type="radio" name="favlang" value="javascript">
+      <label for="JavaScript">JavaScript</label>
+    </div>
+    <label for="signup">Would you like to sign up for more information?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:33]: This is just general text, it should be a `p` element.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <div>
+      <input type="checkbox" name="signup" value="y">Sign me up!
+    </div>
+    <label for="comments">Comments</label>
+    <div>
+      <textarea name="comments" cols="30" rows="10"></textarea>
```
Comment (codingcodymiller) [inline html-forms/index.html:39]: This `textarea` needs an `id` to match the `for` attribute of its `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by dennisliu9** [comment]
html-forms_solution2

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**